db/view/view.cc

```diff
@@ -1772,14 +1772,14 @@ get_view_natural_endpoint(
         }
     }
 
-    auto resolve = [&] (const locator::topology& topology, const locator::host_id& ep, bool is_view) -> const locator::node* {
+    auto resolve = [&] (const locator::topology& topology, const locator::host_id& ep, bool is_view) -> const locator::node& {
```

Comment:
Looks like the `const locator::node*` -> `const locator::node&` fixup got merged into the wrong commit

Reply:
Fixed in v11 ([89cc967](https://github.com/scylladb/scylladb/commit/89cc967f99ce65b260bcd0b037e232fe657fd1d3))